Mathlib/Topology/EMetricSpace/Basic.lean

```diff
@@ -295,3 +295,30 @@ instance [PseudoEMetricSpace X] : EMetricSpace (SeparationQuotient X) :=
       toUniformSpace := inferInstance,
       uniformity_edist := comap_injective (surjective_mk.prodMap surjective_mk) <| by
         simp [comap_mk_uniformity, PseudoEMetricSpace.uniformity_edist] } _
+
+namespace NNReal
+
+open Filter
+
+/-- If `u : ℕ → ℝ` is bounded above and nonnegative, it is also bounded above when regarded as
+  a function to `ℝ≥0`. -/
+theorem bddAbove' {u : ℕ → ℝ} (hu0 : 0 ≤ u) (hu_bdd : BddAbove (Set.range u)) :
+    BddAbove (Set.range fun n : ℕ ↦ (⟨u n, hu0 n⟩ : ℝ≥0)) := by
```

Comment:
This is also a fairly strange statement. Where do you even use it or need it?

Reply:
I have deleted this.